### PR TITLE
Add resources

### PR DESCRIPTION
**Pull request by boylea**
We've decided that stuff is too hidden in the meetup pages, so we're bringing it to the website. The resource information is also overdue for an update, so I'd like some help editing on that page.

* Fixes the readme instruction to actually work for building locally
* Adds resources page
* Improvements to formatting, organization

cc: @Greh (who paired with me on this)

@thursdayb could you edit the content on the resources/index.md page?

### CODE REVIEW

File: resources/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Self Paced Learning Material
+---
+
+There is a lot of learning resources on the internet, it can be overwhelming to figure out will work best for you and where to start. The following are resources that Pyladies PDX members have found useful to them, depending on background.
```
Comment (thursdayb) [inline resources/index.md]: There are

Comment (thursdayb) [reply resources/index.md]: Run on sentence, break up into two

File: resources/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## Tutorials for Beginners, new to programming
+
+If you're brand new, you'll first need to get your computer set up to start programming. We recommend the [Django Girls Installation Instructions](https://tutorial.djangogirls.org/en/installation/#installation). These instructions are great regardless of if you're interested in Django (a web framework). The first paragraph on this page says you can skip if reading from home... don't skip it, the whole point is to get your computer set up. Follow each section up until the "What is Django" chapter. These instructions cover Mac, Linux and Windows, and are frequently updated.
```
Comment (thursdayb) [inline resources/index.md]: regardless of whether or not

File: resources/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+### Science/Data Analysis:
+
+[Software Carpentry](https://swcarpentry.github.io/python-novice-inflammation/) Programming skills for science research
+
+[Data Carpentry](http://www.datacarpentry.org/python-ecology-lesson/) Focus on data organization, cleanup, analysis, and visualization
+
+### Generic Beginner Python
+
+[Think Python](http://greenteapress.com/wp/think-python/) TODO: description (Jules)
```
Comment (thursdayb) [inline resources/index.md]: Address this TODO?

File: resources/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+[Learn Python The Hard Way](https://learnpythonthehardway.org/) General Python intro with a focus on repetition to build a foundation of understanding.  This one costs money.
+
+## Non-beginner resources
+
+For people familiar with Python, or seasoned programmers
```
Comment (thursdayb) [inline resources/index.md]: "seasoned programmers" is confusing here — do you mean "programmers with experience in other languages"?

File: resources/index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+### Web Development
+
+There are a lot of web frameworks in Python, some of the most popular:
```
Comment (thursdayb) [inline resources/index.md]: Consider linking the frameworks in this list to the project's website

File: resources/index.md (hunk 3 above)
Comment (jarrighi) [reply resources/index.md]: Free and highly recommended introduction to Python that also touches on general programming and computer science concepts as well. Each chapter includes both straight-forward and challenging exercises. Leans a bit academic and math oriented, so it's especially good if you're comfortable with high-school/early college level math problems, and ready to start learning by doing. If you're looking for a gentle introduction, try another resource first and come back for this one later.

Comment (jarrighi) [reply resources/index.md]: Let me know if that' too long and I'll try to shave it down.

File: README.md [hunk 6]
```diff
@@ -42,15 +42,17 @@ Make sure this version is being sourced by running: ``` $ ruby --version ``` {{
...
+* Installer Bundler: `$ gem install bundler`
 
-* Install Jekyll by running ``` $ gem install jekyll ``` {{ takes a second to get going }}
+Get the site set up:
 
-* If you haven't already, form the pyladies pdx site repo by pushing "Fork" button, then clone your fork to your machine by running: ``` $ git clone https://github.com/your_github_ursername_here/pyladiespdx.github.com.git ```
+* Fork and clone this repo, then `cd` into the pyladies.github.io directory.
 
-** See the TODOs for things that need to be done; develop locally **
+* Install Jekyll using Bundler: `$ bundle install`
```
Comment (Happy686876) [inline README.md]: Help

**Pull request merged**